packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -179,13 +179,13 @@ describe('blog-env-mssql', () => {
   describe('$queryRaw', () => {
```

Comment:
describe says it's about queryRaw but I see there are tests about queryRawUnsafe also

Maybe split into another describe?

Reply:
I think all tests actually are about `queryRawUnsafe` now, so:
```suggestion
  describe('$queryRawUnsafe', () => {
```